src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -50,18 +44,24 @@ internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings inter
         /// <param name="projectionFillPattern">Projection fill pattern</param>
         /// <param name="cutLinePattern">Cut line pattern</param>
         /// <param name="projectionLinePattern">Projection line pattern</param>
-        /// <returns>OverrideGraphicSettings</returns>
+        /// <param name="transparency">Transparency as integer between 1-100.</param>
+        /// <param name="detailLevel">Detail Level setting, ex: Coarse, Fine etc.</param>
+        /// <param name="halftone">Halftone. True = halftone.</param>
+        /// <returns name="overrides">Override Graphic Settings</returns>
         public static OverrideGraphicSettings ByProperties(
```

Comment:
@ksobon @mjkkirschner agree, there was no option to actually get an element of type FillPatternElement - I would rather suggest to add FillPatternElements to Dynamo instead of allowing any kind of element, because currently they could even plug in a wall element.